### PR TITLE
2020 Rust Event Lineup [Draft]

### PR DESCRIPTION
**Pull request by Zexbe**
This is the draft for the 2020 rust event lineup.

TODO:

- [x] OxidizeConf: https://oxidizeconf.com/

- [x] Rusty Days, https://rusty-days.org/, https://twitter.com/rdconf Fall 2020, https://www.reddit.com/r/rust/comments/esqw30/ann_rust_conference_in_wroc%C5%82aw_poland/

- [x] RustFest Netherlands: Q2 2020. https://netherlands.rustfest.eu/

### CODE REVIEW

**Comment by badboy** [comment]
RustFest Netherlands: Q2 2020.
https://netherlands.rustfest.eu/

**Comment by badboy** [comment]
Rusty Days, https://rusty-days.org/, https://twitter.com/rdconf
Fall 2020

File: posts/2020-01-28-conf-lineup.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+**RustConf**<br>August 20th-21st, 2020
+---
+
+The official [RustConf][conf-site] will be taking place in Portland, Oregan, USA. Last years conference was amazing, and we are excited to see what happens next. See the [website][conf-site], and [Twitter][conf-twitter] for updates as the event date approaches!
+
+[conf-site]: https://rustconf.com/
+[conf-twitter]: https://twitter.com/rustconf
+
+###### Located in *Oregan, USA*
```
Comment (apiraino) [inline posts/2020-01-28-conf-lineup.md]:
```suggestion
###### Located in *Oregon, USA*
```

File: posts/2020-01-28-conf-lineup.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "The 2020 Rust Event Lineup"
+author: Rust Community
+description: "Welcome to 2020; We are excited about the Rust conferences coming up; join us at one near you!"
+---
+
+
+A new decade has started, and we are excited about the Rust conferences coming up. Each conference is an opportunity to learn about Rust, share your knowledge, and to have a good time with your fellow rustacians. Read on to learn more about the events we know about so far.
```
Comment (apiraino) [inline posts/2020-01-28-conf-lineup.md]:
```suggestion
A new decade has started, and we are excited about the Rust conferences coming up. Each conference is an opportunity to learn about Rust, share your knowledge, and to have a good time with your fellow Rustaceans. Read on to learn more about the events we know about so far.
```

File: posts/2020-01-28-conf-lineup.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+**FOSDEM** <br> February 2nd, 2020 
+---
+
+[FOSDEM][fosdem site] stands for the Free and Open Source Developers European Meeting. At this event software developers around the world will meet up, share ideas and collaborate. FOSDOM will be hosting a [Rust devroom][fosdem agenda] workshop that aims to present the features and possibilities offered by Rust, as well as some of the many exciting tools and projects in its ecosystem.
```
Comment (apiraino) [inline posts/2020-01-28-conf-lineup.md]:
```suggestion
[FOSDEM][fosdem site] stands for the Free and Open Source Developers European Meeting. At this event software developers around the world will meet up, share ideas and collaborate. FOSDEM will be hosting a [Rust devroom][fosdem agenda] workshop that aims to present the features and possibilities offered by Rust, as well as some of the many exciting tools and projects in its ecosystem.
```

File: posts/2020-01-28-conf-lineup.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+[latam-site]: https://rustlatam.org/
+[latam-agenda]: https://rustlatam.org/#schedule
+
+###### Located in *Mexico City, Mexico*
+---
+
+**RustConf**<br>August 20th-21st, 2020
+---
+
+The official [RustConf][conf-site] will be taking place in Portland, Oregan, USA. Last years conference was amazing, and we are excited to see what happens next. See the [website][conf-site], and [Twitter][conf-twitter] for updates as the event date approaches!
```
Comment (apiraino) [inline posts/2020-01-28-conf-lineup.md]:
```suggestion
The official [RustConf][conf-site] will be taking place in Portland, Oregon, USA. Last years' conference was amazing, and we are excited to see what happens next. See the [website][conf-site], and [Twitter][conf-twitter] for updates as the event date approaches!
```

File: posts/2020-01-28-conf-lineup.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+**RustLab**<br>October 16th-17th, 2020
+---
+
+[RustLab 2020][lab-site] is a 2 days conference with talks and workshops. The date is set, but the talks are still being planned. We expect to learn more details as we get closer to the date of the conference.
+
+[lab-site]: https://www.rustlab.it
+
+###### Located in *Florence, Italy*
+
+---
```
Comment (apiraino) [inline posts/2020-01-28-conf-lineup.md]: **Rust/GNOME hackfest Spring 2020**<br>Apr, 29th - May, 3rd
---

The goal of these [hackfests][hack-link] is to improve the interactions between Rust and the GNOME libraries. In this hackfest we want to improve the interoperability between Rust and GNOME, improve support of GNOME libraries in Rust and explore solutions to create GObject APIs from Rust. 

[hack-link]: https://wiki.gnome.org/Hackfests/Rust2020

###### Located in **Montréal, Quebec**

File: posts/2020-01-28-conf-lineup.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+For the most up-to-date information on events, visit [timetill]. For meetups, and other events see the [calendar].
```
Comment (XAMPPRocky) [inline posts/2020-01-28-conf-lineup.md]:
```suggestion
For the most up-to-date information on events, visit [timetill.rs][timetill]. For meetups, and other events see the [calendar]. 
```

**Comment by XAMPPRocky** [comment]
@skade Would you mind giving this a review and merging it in? Since it's on the main blog it needs to approved by core team member.

**Comment by skade** [comment]
@XAMPPRocky This is still missing a couple of conferences that are announced by don't have a fixed date yet, e.g. OxidizeConf.

**Comment by Zexbe** [comment]
Conferences have been added.

**Comment by Zexbe** [comment]
@skade I think it has all the conferences now.

**Comment by XAMPPRocky** [comment]
Merging with approval from @skade

**Comment by pietroalbini** [comment]
Reverted the post, can you re-submit it with tomorrow as the publishing date? The reason for the revert is that today is release day, so this post would probably be ignored by the public.

**Comment by Zexbe** [comment]
@pietroalbini Okay new pull request made for tomorrow. https://github.com/rust-lang/blog.rust-lang.org/pull/510

**Pull request merged**